undefined_syms.txt

```diff
@@ -2194,12 +2179,6 @@ D_06001C80 = 0x06001C80;
 // z_en_sa
 D_02005730 = 0x02005730;
 D_02010E20 = 0x02010E20;
```

Comment:
I know is a bit out of scope, but maybe you could take a look at those cutscenes and name them too. This way Saria would be complete :D